dev/docker/docker-compose.yml

```diff
@@ -5,13 +5,14 @@ services:
     environment:
       - GOWAKU-NODEKEY=8a30dcb604b0b53627a5adc054dbf434b446628d4bd1eccc681d223f0550ce67
     command:
-      - --store
-      - --message-db-connection-string=postgres://postgres:xmtp@db:5432/postgres?sslmode=disable
-      - --message-db-reader-connection-string=postgres://postgres:xmtp@db:5432/postgres?sslmode=disable
+      - --store.enable
+      - --store.db-connection-string=postgres://postgres:xmtp@db:5432/postgres?sslmode=disable
+      - --store.reader-db-connection-string=postgres://postgres:xmtp@db:5432/postgres?sslmode=disable
       - --wait-for-db=30s
       # Disable authn until we have reliable support for generating auth tokens
       # - --api.authn.enable
     ports:
+      - 5555:5555
```

Comment:
Are these changes related?

Reply:
No, but they're necessary for the tests to pass. The v2 branch didn't get the fixes for the docker-compose arg updates, I guess.